demos/gallery/samples/geospatial/Choropleth/USA/Pins.js

```diff
@@ -26,15 +26,15 @@ const pins = new Pins()
     .longColumn("longtitude")
     ;
 
-new Layered()
-    .target("target")
+const layered = new Layered()
     .layers([
         graticule,
         usStates,
         pins
     ])
+    .target("target")
     .projection("AlbersUsa")
     .render()
-    .resize()
-    .render()
     ;
+
+layered.resize().render();
```

Comment:
Can you open an issue for this and add a comment here linking to that issue.  (double check it is still needed first...)